rocAL/rocAL_pybind/amd/rocal/plugin/tf.py

```diff
@@ -121,26 +145,33 @@ def __next__(self):
             self.num_bboxes_list = []
             self.loader.get_img_sizes(self.img_size)
             for i in range(self.bs):
-                self.label_2d_numpy = np.reshape(self.labels[i], (-1, 1)).tolist()
+                self.label_2d_numpy = np.reshape(
+                    self.labels[i], (-1, 1)).tolist()
                 self.bb_2d_numpy = np.reshape(self.bboxes[i], (-1, 4)).tolist()
                 self.num_bboxes_list.append(len(self.bboxes[i]))
                 self.label_list.append(self.label_2d_numpy)
                 self.bbox_list.append(self.bb_2d_numpy)
 
             self.target = self.bbox_list
             self.target1 = self.label_list
-            max_cols = max([len(row) for batch in self.target for row in batch])
+            max_cols = max([len(row)
+                           for batch in self.target for row in batch])
             # max_rows = max([len(batch) for batch in self.target])
             max_rows = 100
-            bb_padded = [batch + [[0] * (max_cols)] * (max_rows - len(batch)) for batch in self.target]
-            bb_padded_1 = [row + [0] * (max_cols - len(row)) for batch in bb_padded for row in batch]
+            bb_padded = [
+                batch + [[0] * (max_cols)] * (max_rows - len(batch)) for batch in self.target]
```

Comment:
@fiona-gladwin why have these extra lines been added?

Reply:
The python formatter resulted in this kind of formatting, where it split longer lined into 2 lines